vtex-apps/google-geolocation-resolver
# CHK-317, CHK-319: Initial Setup

#### What is the purpose of this pull request?

Initial setup of the first geolocation provider/resolver.

Similar to vtex-apps/geolocation-graphql-interface/pull/1, the setup is based on [this document](https://docs.google.com/document/d/11DewV9j4qiz_U_tuLsAHUa235oydbcSupIkuVdMi9kg), https://github.com/vtex-apps/graphql-interface-app-example and https://github.com/vtex-apps/search-resolver.

As `places-graphql v0.3.0` already had most of the resolver's logic, this PR is kind of big and despite being the initial setup, it also brings the resolvers from `places-graphql`.

#### How should this be manually tested?

1. [Workspace](https://w0geo--checkoutio.myvtex.com/cart/add?sku=289)
2. Go to the shipping step inside checkout
3. Type `rua nascimento silva, 107` in the input
4. Select the first suggestion. It should display delivery options

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/26108090/96290247-a9162080-0fbc-11eb-9292-14c6ab3e25c5.png)
